packages/hooks/src/useWalletsList.ts

```diff
@@ -66,8 +66,16 @@ const useWalletsList = () => {
     const { is_dark_mode_on } = ui;
     const { getConfig } = useCurrencyConfig();
 
-    const { data: authorize_data, ...rest } = useAuthorize();
-    const { data: balance_data } = useFetch('balance', { payload: { account: 'all' } });
+    const {
+        data: authorize_data,
+        isSuccess: is_authorize_success,
+        isFetching: is_authorize_fetching,
+        ...rest
+    } = useAuthorize();
+    const { data: balance_data } = useFetch('balance', {
```

Comment:
@sergei-deriv since we want to make our hooks dummy and independent from any thing else, it's not a good approach to start fetching data base on sth else outside. In the usage of the hook itself we can apply the latency and how we need to call the hook.